src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -654,10 +660,15 @@
                                         AllowsTransparency="True"
                                         MouseLeave="OnPopupMouseLeave">
             <uicontrols:LibraryToolTipPopup.IsOpen>
-                <Binding ElementName="libraryToolTipPopup"
-                         Path="DataContext"
-                         Mode="OneWay"
-                         Converter="{StaticResource NodeSearchElementVMToBoolConverter}" />
+                <MultiBinding Converter="{StaticResource MultiBoolToBoolConverter}">
+                    <Binding Path="DataContext.ShowTooltipForTopResult"
+                             Mode="OneWay"
+                             RelativeSource="{RelativeSource FindAncestor, AncestorType={x:Type search:SearchView}}"/>
+                    <Binding ElementName="libraryToolTipPopup"
+                             Path="DataContext"
+                             Mode="OneWay"
+                             Converter="{StaticResource NodeSearchElementVMToBoolConverter}" />
+                </MultiBinding>
```

Comment:
Hi @aosyatnik, I think it is worth taking a step back and think about this more. Fundamentally I am wondering why do we bind `LibraryToolTipPopup.IsOpen` to Boolean properties and not showing the tool-tip when mouse is hovering over an item? Or more clearly, can we not do a `libraryToolTipPopup.IsOpen = true` in C# when mouse-over event is fired on the item?

Because of the binding, we now have another `ShowTooltipForTopResult` property, which does not look too right to me, by right showing tool-tip or not should based entirely on whether or not the mouse is hovering over an item. Does that make sense?

Please let me know if I have overlooked anything that led to the decision of using data-binding instead of a mouse event, I may have missed something here. Thanks!